### PR TITLE
Unit test for Alert 

### PR DESCRIPTION
**Pull request by namnguyen20999**
image

### CODE REVIEW

File: frontend/taipy-gui/src/components/Taipy/Alert.spec.tsx [hunk 1]
```diff
@@ -12,49 +12,191 @@
...
+        const linkElement = document.querySelector("link[rel='icon']");
+        if (linkElement) {
+            expect(linkElement.getAttribute("href")).toBe("/test-icon.png");
+        } else {
+            expect(true).toBe(false);
+        }
+        document.head.removeChild(link);
+    });
+
+    it("closes Snackbar on close button click", async () => {
```
Comment (FabienLelaquais) [inline frontend/taipy-gui/src/components/Taipy/Alert.spec.tsx]:
```suggestion
    it("closes alert on close button click", async () => {
```

File: frontend/taipy-gui/src/components/Taipy/Alert.spec.tsx [hunk 2]
```diff
@@ -12,49 +12,191 @@
...
+        );
+        const closeButton = await screen.findByRole("button", { name: /close/i });
+        await userEvent.click(closeButton);
+        await waitFor(() => {
+            const alertMessage = screen.queryByText("Test Alert");
+            expect(alertMessage).not.toBeInTheDocument();
+        });
+    });
+
+    it("Snackbar disappears when alert type is empty", async () => {
```
Comment (FabienLelaquais) [inline frontend/taipy-gui/src/components/Taipy/Alert.spec.tsx]:
```suggestion
    it("Alert disappears when alert type is empty", async () => {
```

**Review by FabienLelaquais — DISMISSED** [review]
Minor renaming stuff. Great job!

**Review by FredLL-Avaiga — DISMISSED** [review]

File: frontend/taipy-gui/src/components/Taipy/Alert.spec.tsx (hunk 2 above)
Comment (namnguyen20999) [reply frontend/taipy-gui/src/components/Taipy/Alert.spec.tsx]: updated

File: frontend/taipy-gui/src/components/Taipy/Alert.spec.tsx (hunk 1 above)
Comment (namnguyen20999) [reply frontend/taipy-gui/src/components/Taipy/Alert.spec.tsx]: updated

**Review by FredLL-Avaiga — APPROVED** [review]
Are you using prettier to format ?

**Review by FabienLelaquais — APPROVED** [review]
👍

**Pull request merged**